articles/quickstart.md

```diff
@@ -1,163 +1,129 @@
 ---
-title: Write a quantum program | Microsoft Docs
-description: Learn how to write a quantum program in Q#. Develop a Bell  State application in Visual Studio.
-author: QuantumWriter
-ms.author: Alan.Geller@microsoft.com 
-ms.date: 12/11/2017
-ms.topic: article
+title: Write your first quantum program
+description: Learn how to write a quantum program in Q#. Develop a Bell State application using the Quantum Development Kit (QDK)
+author: natke
+ms.author: nakersha 
+ms.date: 10/07/2019
+ms.topic: tutorial
 uid: microsoft.quantum.write-program
 ---
-# Writing a Quantum Program
+# Write your first quantum program
+
+Learn how to write a quantum program using the Microsoft QDK. You will write an application to demonstrate the simplest form of quantum entanglement, also known as **Bell State**.
```

Comment:
```suggestion
Learn how to write a quantum program using the Microsoft QDK. You will write an application to demonstrate the simplest form of quantum entanglement, a **Bell state**.
```

Reply:
Feel free to play with the phrasing, but basically trying to avoid implying that the term "Bell state" is just a moniker for "the simplest form of quantum entanglement". Might be worth going with "one of the simplest forms..." as well, just to avoid being too concrete. 

(also, changed to "a" because there are actually four of them)